config/node-commonjs.js

```diff
@@ -8,7 +8,6 @@ const filePath = (file) => path.join(process.cwd(), 'dist', file);
 
 // Main entry
 fs.copyFileSync(filePath('index.js'), filePath('commonjs.js'));
-fs.copyFileSync(filePath('index.js.map'), filePath('commonjs.js.map'));
```

Comment:
No point in copying the maps AFAIK, we don't update the source map comment, i.e., `commonjs.js` still has `//# sourceMappingURL=index.js.map`.

Could fix that,  but I don't see a real need to?